src/Files.App/UserControls/TabBar/TabBar.xaml.cs

```diff
@@ -51,6 +55,14 @@ public bool AllowTabsDrag
 		public Rectangle DragArea
 			=> DragAreaRectangle;
 
+		private GridLength _titleBarWidth;
+
+		public GridLength TitleBarWidth
```

Comment:
```suggestion
		public GridLength TitleBarRightInsetWidth
```

Reply:
It is possible to change, but a gap is already added to the value.